code/__DEFINES/flags.dm

```diff
@@ -126,6 +126,8 @@ GLOBAL_LIST_INIT(bitflags, list(1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 204
 #define PERSISTENT_ENGRAVINGS (1<<14)
 /// Mobs that die in this area don't produce a dead chat message
 #define NO_DEATH_MESSAGE (1<<15)
+/// This area is not considered to be hull breachable, used by floorbots so they dont close up all of space.
+#define NOT_HULLBREACHABLE (1<<15)
```

Comment:
`NOT_HULL_BREACHABLE`, but preferably find a way to word this in the positive because `if(!(flags & NOT_))` is weird to read